src/app/app-routing.module.ts

```diff
@@ -2,14 +2,16 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 import { AppRoutePath } from './enums/app-route-path.enum';
-import { CoursesPageComponent } from './pages/courses-page/courses-page.component';
-import { LoginPageComponent } from './pages/login-page/login-page.component';
+import { LoginPageComponent } from './pages/auth/login-page/login-page.component';
+import { CourseAddPageComponent } from './pages/courses/course-add-page/course-add-page.component';
+import { CoursesPageComponent } from './pages/courses/courses-page/courses-page.component';
 
 const routes: Routes = [
+  { path: AppRoutePath.CourseAdd, component: CourseAddPageComponent },
```

Comment:
I'd suggest making it a part of CoursesRouting  module.

Reply:
anyway, it will be within next home works I guess..

Reply:
I think that if we consider to create new routing module, there should be component for holding new `<router-outlet></router-outlet>`. I don't see appropriate components in current project. Also I don't want to create new component just for holding `<router-outlet></router-outlet>`. This approach looks a bit redundant for me. I'll create separate `*.route.ts` files for every entity for now.